### PR TITLE
Update twine-cert.txt

### PR DESCRIPTION
**Pull request by mdopf**
Added English text

### CODE REVIEW

File: twine-cert.txt [hunk 1]
```diff
@@ -1,348 +1,432 @@
...
 [vaccination_fourth_onboarding_page_button_title]
-de = Alles klar
+de = OK
 tags = common
 [vaccination_privacy_policy_page_button_title]
 de = Schließen
+en = Close
 tags = common
 [[start Screen]]
-[dialog_start_screen_title]
```
Comment (timokoenig) [inline twine-cert.txt:65]: Why do you remove this?

File: twine-cert.txt [hunk 2]
```diff
@@ -1,348 +1,432 @@
...
 de = Irland
+en = Ireland
 tags = common
 [CH]
 de = Schweiz
+en = Switzerland
 tags = common
 [certificate_check_validity_selection_country_note]
-de = Sie finden Ihr Reiseland nicht? In einem mehrstufigen Prozess werden nach und nach mehr Länder verfügbar gemacht. \n\nMehr Informationen finden Sie in den Häufigen Fragen und unter #https://reopen.europa.eu::https://reopen.europa.eu#.
+de = Sie finden Ihr Reiseland nicht? In einem mehrstufigen Prozess werden nach und nach mehr Länder verfügbar gemacht. \n\nMehr Informationen finden Sie in den #Häufigen Fragen::Häufigen Fragen# und unter #https://reopen.europa.eu::https://reopen.europa.eu#.
```
Comment (timokoenig) [inline twine-cert.txt:314]: Please do not use those links at the moment `#Häufigen Fragen::Häufigen Fragen#`

File: twine-cert.txt [hunk 3]
```diff
@@ -442,411 +548,543 @@ tags = common
...
 de = Aktuell verwendetes Zertifikat
+en = Certificate currently in use
 tags = common
 [certificates_overview_vaccination_certificate_title]
 de = Impfzertifikat
+en = Vaccination certificate
 tags = common
 [certificates_overview_vaccination_certificate_message]
 de = Impfung %d von %d
+en = Vaccine dose 1 of 2
```
Comment (timokoenig) [inline twine-cert.txt:647]: Please use same format as German translation

File: twine-cert.txt [hunk 4]
```diff
@@ -442,411 +548,543 @@ tags = common
...
 de = Open-Source-Lizenzen
+en = Open-source licences
 tags = common
 [app_information_message_open_source]
 de = Die App verwendet die folgenden Open-Source-Bibliotheken:
+en = The app uses the following open-source libraries:
 tags = common
 [app_information_version_label]
 de = Version %@
+en = Version %d
```
Comment (timokoenig) [inline twine-cert.txt:1089]: Use same format as German translation

**Review by timokoenig — CHANGES_REQUESTED** [review]

File: twine-cert.txt [hunk 5]
```diff
@@ -76,273 +96,353 @@ de = Aktualisieren
...
 de = Im Reiseland ungültig
+en = Invalid in this country
 tags = common
 [certificate_check_validity_result_not_testable]
 de = Nicht vollständig prüfbar
+en = Full check not possible
 tags = common
 [certificate_check_validity_note]
 de = Beachten Sie, dass sich die Einreiseregeln ändern können. Prüfen Sie daher die Zertifikate mindestens 48 Stunden vorher. Es können in einzelnen Regionen oder bei Ihrer Rückkehr weitere Regeln oder Einschränkungen gelten. Zudem ist es möglich, dass weitere Dokumente für eine Einreise notwendig sind.\n\nMehr Informationen finden Sie in den Häufigen Fragen und unter #https://reopen.europa.eu::https://reopen.europa.eu#.
+en = Please be aware that travel regulations can change. You should check your certificates at least 48 hours before you travel. Different regulations or restrictions might apply in some regions or when you return. You might also need additional documents to travel to a different country. \n\nYou can find more information in the frequently asked questions and on the #Re-open EU::Re-open EU# information page.
```
Comment (timokoenig) [inline twine-cert.txt:395]: Please use URL like in the German translation

Comment (mdopf) [reply twine-cert.txt]: Alright. I changed it.

**Comment by timokoenig** [comment]
LGTM

**Pull request merged**